From: intrigeri <intrigeri@boum.org>
Date: Mon, 29 Jan 2018 07:40:37 +0000
Subject: AppArmor: grant access to mostly innocuous stuff plugin-container
 tries to read.

Same rationale as commit 68f502c3fbb754742cd23967cf30038ff6ce799a.
---
 apparmor/torbrowser.Browser.plugin-container | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/apparmor/torbrowser.Browser.plugin-container b/apparmor/torbrowser.Browser.plugin-container
index 5fd3efa..4ee65ec 100644
--- a/apparmor/torbrowser.Browser.plugin-container
+++ b/apparmor/torbrowser.Browser.plugin-container
@@ -10,7 +10,6 @@ profile torbrowser_plugin_container {
   # bellow.
   # #include <abstractions/audio>
   # /etc/asound.conf r,
-  # owner @{PROC}/@{pid}/fd/ r,
   # owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/tmp/mozilla-temp-* rw,
 
   signal (receive) set=("term") peer=torbrowser_firefox,
@@ -29,6 +28,9 @@ profile torbrowser_plugin_container {
   /etc/mime.types r,
   /usr/share/applications/gnome-mimeapps.list r,
 
+  /dev/shm/ r,
+
+  owner @{PROC}/@{pid}/fd/ r,
   owner @{PROC}/@{pid}/mountinfo r,
   owner @{PROC}/@{pid}/stat r,
   owner @{PROC}/@{pid}/status r,
